fineract-provider/src/main/java/org/apache/fineract/infrastructure/report/service/PentahoReportingProcessServiceImpl.java

```diff
@@ -183,6 +189,11 @@ private void addParametersToReport(final MasterReport report, final Map<String,
             var tenantUrl = toJdbcUrl(fineractProperties.getTenant().getProtocol() + ":" + fineractProperties.getTenant().getSubprotocol(),
                     tenantConnection.getSchemaServer(), tenantConnection.getSchemaServerPort(), tenantConnection.getSchemaName(),
                     tenantConnection.getSchemaConnectionParameters());
+            if (isPentahoReportEngineRunOnPostgres) {
+                tenantUrl = toJdbcUrl(fineractProperties.getTenant().getProtocol() + ":postgresql", tenantConnection.getSchemaServer(),
```

Comment:
Instead of hardcoding "postgresql" you could use `fineractProperties.getTenant().getSubprotocol()`, so you would need to set it in the application properties.

Reply:
This has been fixed. Thanks for pointing it out